tasks/setup-Amazon.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Enable Amazon Linux Extras Docker repository
+  command: amazon-linux-extras enable docker  # noqa 301
```

Comment:
Why enable instead of install?

I tried enable, doesn't work.... 

What is the meaning of the comment?

It's also missing `-y` to perform the install automatically instead of waiting for user input.

Reply:
I've just enabled because it will be installed in later [here](https://github.com/geerlingguy/ansible-role-docker/blob/e697edb53330d3ba3f9cc94978fc0326639cfc2d/tasks/main.yml#L18)

Reply:
does that work? (does it resolve all the dependencies?)

I run install manually now, so not sure.

Reply:
Yes, it does

Reply:
Good to hear, thank you for the feedback